src/GitHubCreateMergePRs/config.xml

```diff
@@ -8,8 +8,6 @@
     <!-- Roslyn servicing branches (flowing between releases) -->
     <merge from="dev15.9.x" to="release/dev16.11-vs-deps" />
     <merge from="release/dev16.11-vs-deps" to="release/dev17.6" />
-    <merge from="release/dev17.6" to="release/dev17.8" />
-    <merge from="release/dev17.8" to="release/dev17.10" />
```

Comment:
Did we want to kill the 17.8->17.10 flow as well?

Reply:
We do for now at least since we don't want this PR to flow forward https://github.com/dotnet/roslyn/pull/75348.